### PR TITLE
Added pyrobuf

### LINKED ISSUE #6423: Package pyrobuf
Would be good to have a package of [`pyrobuf`]( https://pypi.org/project/pyrobuf/ ), a fast Cython implementation of Google's Protobuf.

### PR DESCRIPTION
**Pull request by hmaarrfk**
Fixes #6423

### CODE REVIEW

File: recipes/pyrobuf/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - pip
+    - msinttypes  # [win]
+    - jinja2 >=2.8
+    - cython >=0.23
+    - pytest-runner
```
Comment (marcelotrevisani) [inline recipes/pyrobuf/meta.yaml]: Is ``pytest-runner`` really necessary to build the package? Could you please confirm that?

**Comment by hmaarrfk** [comment]
image

File: recipes/pyrobuf/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipes/pyrobuf/meta.yaml]: Usually if it is a dependency yes. Though it appears to be verified below.

**Comment by hmaarrfk** [comment]
@jakirkham, want to be comainter? I was mostly gaining experience in packaging.

File: recipes/pyrobuf/meta.yaml (hunk 1 above)
Comment (hmaarrfk) [reply recipes/pyrobuf/meta.yaml]: I was surprised too, I actually tried to build it without it first, but it turned out to be important.

File: recipes/pyrobuf/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  run:
+    - jinja2 >=2.8
+    - cython >=0.23
```
[the remark this answers is not in the document]
Comment (hmaarrfk) [reply recipes/pyrobuf/meta.yaml]: Seems like they template alot, and probably compile on the fly? would they need a c++ compiler too?

**Pull request merged**